class-pikpay.php

```diff
@@ -596,21 +602,21 @@ public function generate_form($order_id)
 
             'language' => $this->form_language,
             'transaction_type' => $transaction_type,
-            'authenticity_token' => $this->pikpayauthtoken,
+            'authenticity_token' => $this->monri_auth_token,
```

Comment:
@4050350194 rename to `monri_authenticity_token`. Renaming keys will greatly reduce potential miss-understandings.

Reply:
Resolved by commit ID: 2ecebc1d36ff40711be89c5071d93220744b53a9.